hack/observability/opentelemetry/values.yaml

```diff
@@ -0,0 +1,69 @@
+mode: "deployment"
+
+config:
+  receivers:
+    # using default configs for otlp
+    jaeger: null # disable Jaeger receiver
+    prometheus: null # disable Prometheus receiver
+    zipkin: null #  disable Zipkin receiver
+
+  # (2) Processors
+  processors:
+    memory_limiter:
+      limit_percentage: 50
+      check_interval: 1s
+      spike_limit_percentage: 30
+    batch:
+      send_batch_size: 8192
+
+  # (3) exporter
+  exporters:
+    jaeger:
+      endpoint: jaeger-all-in-one:14250
+      tls:
+        insecure: true
+    #export to Cloud Trace on GCP
+    googlecloud:
```

Comment:
Refer to [GCP Cloud trace exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter) implementation for OTEL Collector